src/core/agent/services/multiSigService.ts

```diff
@@ -975,8 +985,25 @@ class MultiSigService extends AgentService {
   async multisigAdmit(
     multisigSignifyName: string,
     notificationSaid: string,
-    timeAdmit?: string
+    allSchemaSaids?: any[],
```

Comment:
`schemaSaids: string[]` - we need to stop using `?` and if statements for mandatory fields. This should be passed in both places where we admit the credential with a multi-sig